backend/src/test/java/ai/verta/modeldb/TestsInit.java

```diff
@@ -97,13 +97,13 @@ protected static void init() throws Exception {
     testConfig = TestConfig.getInstance();
     handleExecutor = FutureGrpc.initializeExecutor(testConfig.getGrpcServer().getThreadCount());
     // Initialize services that we depend on
-    services = ServiceSet.fromConfig(testConfig, testConfig.artifactStoreConfig);
-    authService = services.authService;
-    // Initialize data access
-    daos = DAOSet.fromServices(services, testConfig.getJdbi(), handleExecutor, testConfig);
-    App.migrate(testConfig.getDatabase(), testConfig.migrations);
+    //    services = ServiceSet.fromConfig(testConfig, testConfig.artifactStoreConfig);
+    //    authService = services.authService;
+    //    // Initialize data access
+    //    daos = DAOSet.fromServices(services, testConfig.getJdbi(), handleExecutor, testConfig);
+    //    App.migrate(testConfig.getDatabase(), testConfig.migrations);
 
-    App.initializeBackendServices(serverBuilder, services, daos, handleExecutor);
+    //    App.initializeBackendServices(serverBuilder, services, daos, handleExecutor);
```

Comment:
please delete if not needed, rather than leaving comment cruft.

Reply:
Fixed